blazor/treeview/navigation.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: post
+title: Navigation in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about the navigation in Syncfusion Blazor TreeView component and much more.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Navigation URL binding in Blazor TreeView Component
+
+The TreeView component's **NavigateUrl** property is used to navigate from one page to other pages with TreeView node selection.

```

Comment:
Use the **NavigateUrl** property to specify the URL to navigate to when the tree node is selected.

Reply:
Done.